pom.xml

```diff
@@ -18,15 +18,15 @@
         <maven.checkstyle.version>3.1.1</maven.checkstyle.version>
         <maven.compiler.source>11</maven.compiler.source>
         <maven.compiler.target>11</maven.compiler.target>
-        <log4j.version>2.13.3</log4j.version>
+        <log4j.version>2.17.1</log4j.version>
         <slf4j-simple.version>1.6.2</slf4j-simple.version>
         <kafka.version>3.0.0</kafka.version>
         <opentracing-kafka.version>0.1.15</opentracing-kafka.version>
         <jaeger.version>1.1.0</jaeger.version>
         <strimzi-oauth-callback.version>0.8.1</strimzi-oauth-callback.version>
-        <vertx-core.version>4.1.0</vertx-core.version>
-        <netty.version>4.1.66.Final</netty.version>
-        <jackson.version>2.12.4</jackson.version>
+        <vertx-core.version>4.2.2</vertx-core.version>
```

Comment:
Why not go for 4.2.3 when you update it?

Reply:
I didn't saw it at first, good catch. Thanks :)